ivadomed/main.py

```diff
@@ -319,7 +321,8 @@ def cmd_train(context):
                   mt_metrics.specificity_score,
                   mt_metrics.intersection_over_union,
                   mt_metrics.accuracy_score]
-
+    att_maps = list()
+    int_imgs = list()
```

Comment:
Could you please add a short comment here? something like "int_imgs stores XX"

Reply:
Oups this shouldn't be here, it is not used!